src/azul/plugins/repository/dss/__init__.py

```diff
@@ -240,6 +240,9 @@ def update(self,
             query = urllib.parse.parse_qs(location.query, strict_parsing=True)
             self.token = one(query['token'])
             self.replica = one(query['replica'])
+            # REVIEW: The commit that touches this line also removes another one,
+            #         including a comment that seemed important. Please request
+            #         PL to discuss this.
```

Comment:
Agreed at PL to investigate the meaning of the removed comment.

Reply:
[The only test covering this code sets the file's DRS URI to `None`](https://github.com/DataBiosphere/azul/blob/71a10b082c43e7b5c8f87c024091211511c11c33/test/service/test_repository_files.py#L279), so the assignment has no effect.

Reply:
The test is very old and the history is hard to track, but AFAICT a null DRS URI has always been used here. If the mock file is edited to use anon-null DRS URI, and the assignment is removed, the test fails because the retry URL unexpectedly contains the `drsUri` request parameter.